example/scalalib/native/1-simple/build.mill

```diff
@@ -0,0 +1,56 @@
+//// SNIPPET:BUILD
+package build
+import mill._, scalalib._, scalanativelib._
+
+object `package` extends RootModule with ScalaNativeModule {
+  def scalaVersion = "2.13.11"
+  def scalaNativeVersion = "0.5.5"
+
+  // You can have arbitrary numbers of third-party dependencies
+  def ivyDeps = Agg(
+    ivy"com.lihaoyi::mainargs::0.7.6"
+  )
+
+  object test extends ScalaNativeTests {
+    def ivyDeps = Agg(ivy"com.lihaoyi::utest::0.8.4")
+    def testFramework = "utest.runner.Framework"
+  }
+}
+//// SNIPPET:END
+//
+//
+//// SNIPPET:DEPENDENCIES
+//
+//
+/** Usage
+
+
+> ./mill nativeLink  # Build and link native binary
```

Comment:
Let's use `mill show nativeLink` to show people the binary that is generated, and also show `mill run` and `mill test`